test/Microsoft.DotNet.Cli.Utils.Tests/GivenAppThrowingException.cs

```diff
@@ -28,7 +28,7 @@ public void ItShowsStackTraceWhenRun()
             string msg1 = "Unhandled Exception: AppThrowing.MyException: "
                 + "Exception of type 'AppThrowing.MyException' was thrown.";
             string msg2 = "at AppThrowing.MyException.Main(String[] args)";
-            new DotnetCommand(DotnetUnderTest.WithBackwardsCompatibleRuntimes)
```

Comment:
We don't have back compact test any more? Or it is moved to tools set?

Reply:
We do, but now we don't need a separate SDK containing the backwards compatible runtime. We don't need a separate one because we don't package the SDK in here anymore. It is all together under bin/2/dotnet.